ShopApp/App/Modules/Main/Controllers/ProductDetailsViewController.swift

```diff
@@ -280,4 +281,16 @@ class ProductDetailsViewController: BaseViewController<ProductDetailsViewModel>,
         productDetailsViewController.productId = product.id
         navigationController.pushViewController(productDetailsViewController, animated: true)
     }
+    
+    // MARK: - QuantityDropDownViewDelegate
+    
+    func quantityDropDownView(_ view: QuantityDropDownView, didSelectMoreWith quantity: Int) {
+        showQuantityAlert(with: quantity) { [weak self] text in
```

Comment:
Add tests for this method

Reply:
We can't check if alert controller was presented